lib/CommandImplementations.ts

```diff
@@ -542,6 +542,62 @@ function formatVaccineData(data: any, population: number, embed: MessageEmbed) {
     );
 }
 
+export async function handleStock(msg: Message, args: string[]) {
+    if (args.length === 0) {
+        msg.channel.send("You need to include a ticker. Example: $stock IBM");
+        return;
+    }
+
+    const [ ticker ] = args;
+
+    const res = await fetch(`https://www.alphavantage.co/query?function=GLOBAL_QUOTE&symbol=${ticker.toUpperCase()}&apikey=${config.stockApiKey}`);
+    const stockData = await res.json();
```

Comment:
We need to check if a response is actually retrieved and warn the user that their ticker is invalid if not.